GSA-TTS/identity-handbook
# Update Translation Process docs

## 🛠 Summary of changes
- Update content to include Chinese (Simplified) as an available language Login.gov currently supports
- Updated link text and link to point at [Content Translation Process SOP](https://docs.google.com/document/d/16MacAb1WKiQJJ634Cpjg8WkziiyMJMopA07rUpHP2Vs/edit?tab=t.0#heading=h.l6bljy4rjlyu) rather than deprecated [Login.gov Translation Process](https://docs.google.com/document/d/1-wNXxyvxrsUeHkMOfhBpoSTCTZULOXNlCkBdNxiLa3c/edit?tab=t.0#heading=h.w0c3f0nrg6gp)  

## Testing Plan
1. Start server, inspect page for content updates
2. Ensure updated link works

## 👀 Screenshots

| before | after |
| - | - |
| ![Screenshot 2024-12-10 at 11 56 10 AM](https://github.com/user-attachments/assets/5b0ba40a-bcda-433e-aa1f-654d3dc4f796) | ![Screenshot 2024-12-10 at 12 04 16 PM](https://github.com/user-attachments/assets/6b89fa10-020c-445b-a44b-a516054bcfbe) |
